frappe_whatsapp/frappe_whatsapp/doctype/whatsapp_notification/whatsapp_notification.py

```diff
@@ -311,3 +311,20 @@ def template():
     doc.mobile_no = "+919741094468"
     notification.send_template_message(doc)
 
+
+
+@frappe.whitelist()
+def call_trigger_notifications():
+    """Trigger notifications."""
+    try:
+        # Assuming trigger_notifications is defined elsewhere in the file
+        if 'trigger_notifications' in globals():
```

Comment:
The function is already defined in the same file. There is not need to check in globals

Reply:
also, there is no need for a separate function you can whitelist `trigger_notifications` and use the same function.